rules/windows/process_creation/win_non_priv_reg_or_ps.yml

```diff
@@ -0,0 +1,44 @@
+title: Non-privileged Usage of Reg or Powershell
+id: 8f02c935-effe-45b3-8fc9-ef8696a9e41d
+description: Search for usage of reg or Powershell by non-priveleged users to modify service configuration in registry
+status: experimental
+author: Teymur Kheirkhabarov (idea), Ryan Plas (rule), oscd.community
+date: 2020/10/05
+references: 
+    - https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-20-638.jpg
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    integrity_level:
+        IntegrityLevel: Medium
+    reg:
+        CommandLine|contains|all:
+            - reg
+            - add
+    powershell:
+        CommandLine|contains: powershell
+        CommandLine|contains:
+            - set-itemproperty
+            - " sp "
+            - new-itemproperty
```

Comment:
Hello @ryanplasma !

Thank you for your contribution (: 
Could you please tell me what you've tried to achieve in this section? Maybe I can come up with some solution

Reply:
So the idea is that it's looking for ('*powershell*' AND (*set-itemproperty* OR * sp * OR *new-itemproperty*)).  The other idea I had for this was to split it up into `powershell_1` and `powershell_2` and then put the logic in the condition field but there might be a better way so I'm open to any feedback or ideas you have.

Reply:
I believe that your idea with "`powershell_1` and `powershell_2`" fits best in that scenario (: please update your rule in this way and we will pass your PR for merge.

thank you very much for your work!

Reply:
@yugoslavskiy thanks for your help!
@ryanplasma we'll be waiting for you to apply the suggested changes, and I think the PR will be ready (: thanks again for your time and work!